lib/fastlane/plugin/fueled/actions/install_profiles.rb

```diff
@@ -9,12 +9,14 @@ class InstallProfilesAction < Action
       def self.run(params)
         folder = params[:folder]
         UI.message("Crawling #{folder}...")
-        Dir.children(folder).each do |profile|
-          UI.message("Found #{profile}")
-          other_action.install_provisioning_profile(
-            path: File.join(folder, profile)
-          )
-        end
+        Dir.children(folder)
```

Comment:
Should we check if the folder exists?
from the doc [here](https://ruby-doc.org/core-3.1.0/Dir.html#method-c-children):
> Will raise a [SystemCallError](https://ruby-doc.org/core-3.1.0/SystemCallError.html) if the named directory doesn't exist.

Reply:
The folder being a user input, it's fine failing if it does not exist.

Reply:
Can we notify via logging in case there are no mobile provisions installed? Might seem like a silent fail.

Reply:
We could, however the failure happening right after will let the developer know that he's missing the provisioning profile. Will look into adding a warning in a next PR.